### DISCUSSION TITLE
strange modularity value for the karate club network

### DISCUSSION THREAD

**Discussion by mumon** [sknetwork-team/scikit-network#543] [General]
Dear scikit-network team, I was playing with the Louvain algorithm and I realized, even from your documentation, that the optimal modularity value you get for the Karate Club case is 0.688034188034188. This is a very strange value from what I learned in the literature. At my best knowledge, the maximum you can get with Louvain is around 0.42, well below your result. I also checked the community structure you can get with your scikit-network Louvain and computed the modularity with the standard function in the 'community' module of Networkx, obtaining something around 0.42. So it seems that the final partition is kind of right, but your 'get_modularity' produce a wrong(?) value. 
Can you kindly help me in understanding what is going on?
All the best,   alessandro

**Comment by tbonald** [comment]
Thank you very much for the notice. This has just been corrected (see the Develop branch).
It will be available in the next release (coming soon).
All the best!
